src/test/java/de/tum/in/www1/artemis/config/TheiaConfigurationTest.java

```diff
@@ -0,0 +1,41 @@
+package de.tum.in.www1.artemis.config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import de.tum.in.www1.artemis.AbstractSpringIntegrationIndependentTest;
+import de.tum.in.www1.artemis.domain.enumeration.ProgrammingLanguage;
+
+class TheiaConfigurationTest extends AbstractSpringIntegrationIndependentTest {
+
+    @Autowired
+    private TheiaConfiguration theiaConfiguration;
+
+    @Test
+    void testAutowired() {
+        assertThat(theiaConfiguration).isNotNull();
+    }
+
+    @Test
+    void testAmountOfLanguageImages() {
+        assertThat(theiaConfiguration.getImagesForAllLanguages()).hasSize(2);
+    }
+
+    @Test
+    void testFlavorsForLanguage() {
+        Map<String, String> images = theiaConfiguration.getImagesForLanguage(ProgrammingLanguage.valueOf("JAVA"));
```

Comment:
```suggestion
        Map<String, String> images = theiaConfiguration.getImagesForLanguage(ProgrammingLanguageJAVA);
```

Reply:
I used this on purpose to showcase the conversion of Strings (coming from the client) to the Server-Side